tutorials/abap-environment-business-partner/abap-environment-business-partner.md

```diff
@@ -22,18 +23,18 @@ author_profile: https://github.com/niloofar-naseri
   - Return business partner id after creation
 
 Create all ABAP artifacts with namespace `Z...` for local development.
-In this tutorial, wherever `XXX` appears, use a number (e.g. `000`).
+In this tutorial, wherever `XXX` appears, use a number e.g. `000`.
```

Comment:
Please add comma before e.g.
Consider using "for example" instead of "e.g." for better readability.

Reply:
how can I use ()?
because I got just syntax error (not balanced) by using (e.g.`000`)